ProcessesApi/V1/Helpers/SoleToJointHelpers.cs

```diff
@@ -88,7 +88,18 @@ public static void ValidateHasNotifiedResident(this ProcessTrigger processReques
             ValidateFormData(formData, new List<string>() { SoleToJointFormDataKeys.HasNotifiedResident });
 
             if (formData.ContainsKey(SoleToJointFormDataKeys.Reason))
-                _eventData = SoleToJointHelpers.CreateEventData(formData, new List<string> { SoleToJointFormDataKeys.Reason });
+                _eventData = CreateEventData(formData, new List<string> { SoleToJointFormDataKeys.Reason });
```

Comment:
Can you make it return eventData instead of adding a new property to the helper?